docs/notebooks/getting_started.ipynb

```diff
@@ -0,0 +1,2465 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    adata.layers["raw"] = sparse.csr_matrix((np.exp(adata.X.A) - 1).astype(int))
</pre><p>This looks like black magic to a newbie. Maybe add a note that this isn't really part of the tutorial? </p><p> </p><p>We could also consider providing the appropriately processed anndata object as a separate example dataset. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/scverse-tutorials/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>